docs/rule/require-valid-alt-text.md

```diff
@@ -28,16 +27,18 @@ This rule **forbids** the following:
 <img src="bar" alt="Image of me at a bar!" />
 <img src="baz" alt="Picture of baz fixing a bug." />
 <img src="b52.jpg" alt="52" />
+<img src="foo" alt="foo as a banana" role="presentation">
 ```
 
 This rule **allows** the following:
 
 ```hbs
 <img src="rwjblue.png" alt="A man standing in front of a room of people, giving a presentation about Ember.">
 <img src="foo" alt="YourCompany Home Page" />
-<img src="bar" aria-hidden alt="Picture of me taking a photo of an image" /> // Will pass because it is hidden.
```

Comment:
Hehe, we really should have a rule to enforce this one too 😝 (no-invalid-aria-attributes :wink: ).

Reply:
Yes, probably should :) I'll write a new issue.